lib/ansible/modules/cloud/ovirt/ovirt_cluster.py

```diff
@@ -214,6 +214,17 @@
             - "C(Note:)"
             - "This is supported since oVirt version 4.1."
         version_added: 2.4
+    external_network_providers:
+        description:
+            - "List of references to the external network providers available
+               in the cluster. If the automatic deployment of the external
+               network provider is supported, the networks of the referenced
+               network provider are available on every host in the cluster."
+            - "External network provider is described by following dictionary:"
+            - "C(name) - Name of the external network provider. Either C(name)
+               or C(id) is reuqired."
+            - "C(id) - ID of the external network provider. Either C(name) or
+               C(id) is reuqired."
```

Comment:
please add `version_added: 2.5` that's why the build fails

Reply:
Thanks for the helpful hint!